src/test/java/core/basesyntax/serviceimpl/ReportWriterServiceImplTest.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.serviceimpl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import core.basesyntax.service.ReaderService;
+import core.basesyntax.service.ReportWriterService;
+import java.util.List;
+import org.junit.Test;
+
+public class ReportWriterServiceImplTest {
+    private static final String VALID_PATH = "src/test/java/resources/testreport.csv";
+    private static final String report = "fruit,quantity" + System.lineSeparator()
+            + "banana,152" + System.lineSeparator() + "apple,90";
+    private final ReportWriterService reportWriterService = new ReportWriterServiceImpl();
+    private final ReaderService readerService = new ReaderServiceImpl();

```

Comment:
![image](https://user-images.githubusercontent.com/117547096/219877726-f3d7c08d-bcee-4278-84e1-e583713f077b.png)

When you are testing ReportWriterService - you should not use ReaderService. See common mistakes
```suggestion
```